src/legacy/dialog/test_dialog/dialog_test.go

```diff
@@ -0,0 +1,96 @@
+package test_dialog
+
+import (
+	"testing"
+	"time"
+
+	"github.com/noxworld-dev/opennox-lib/platform"
+	"github.com/noxworld-dev/opennox-lib/strman"
+	"github.com/noxworld-dev/opennox/v1/legacy/client/audio/ail"
+	"github.com/noxworld-dev/opennox/v1/legacy/common/alloc/handles"
+	"github.com/noxworld-dev/opennox/v1/legacy/dialog"
+	"github.com/noxworld-dev/opennox/v1/legacy/timer"
+	"github.com/shoenig/test/must"
+)
+
+func platformTicks() uint64 {
+	return uint64(platform.Ticks() / time.Millisecond)
+}
+
+func Setup() {
```

Comment:
```suggestion
func Setup(t testing.TB) {
```
This way you can call:
```go
t.Cleanup(handles.Release)
```
And it will automatically run this function at the end of the test.